nixos/modules/services/networking/coturn.nix

```diff
@@ -0,0 +1,328 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  cfg = config.services.coturn;
+  pidfile = "/var/run/turnserver.pid";
```

Comment:
Only cosmetic since `/var/run/` is usually a symlink to `/run/`, but using `/run/` is preferable.